lib/UmdMainTemplatePlugin.js

```diff
@@ -109,8 +110,10 @@ UmdMainTemplatePlugin.prototype.apply = function(compilation) {
 
 		return new ConcatSource(new OriginalSource(
 			"(function webpackUniversalModuleDefinition(root, factory) {\n" +
+			"   //" + (this.auxiliaryComment && typeof this.auxiliaryComment === 'string' ? this.auxiliaryComment : (this.auxiliaryComment.commonjs2 || "")) + "\n" +
```

Comment:
I attempted to implement something like this: 

```
		function getAuxiliaryCommentFor(auxiliaryCommentKey) {
			var comment; 
			if (this.auxiliaryComment) {
				if (typeof this.auxiliaryComment === 'string') {
					comment = "   //" + this.auxiliaryComment + "\n";
				} else if (typeof this.auxiliaryComment === 'object') {
					if (this.auxiliaryComment.hasOwnProperty(auxiliaryCommentKey)) {
						comment = "  //" + this.auxiliaryComment[auxiliaryCommentKey] + "\n";
					} else {
						comment = "";
					}
				}
			}
			return comment;
		}
```

and replace those lines with `getAuxiliaryCommentFor('commonjs2');` but keep seeing jsbeautify errors. Any ideas?